1.2.c

```diff
@@ -0,0 +1,37 @@
+#define _CRT_SECURE_NO_WARNINGS
+#include <stdio.h>
+#include <math.h>
+
+int main()
+{
+	printf ("Input a: ");
+	int a;
+	scanf ("%d", &a);

```

Comment:
```suggestion
	scanf("%d", &a);

```
Лишний пробел (здесь и в нескольких других вызовах)